src/test/kotlin/com/wgslfuzz/semanticspreservingtransformations/RectsTransformReduceTests.kt

```diff
@@ -30,9 +29,12 @@ class RectsTransformReduceTests : TransformReduceTests() {
         assertNotEquals(filenameNoExtension, "")
     }
 
-    @Disabled("TODO(https://github.com/mc-imperial/wgsl-fuzz/issues/169)")
     @Test
     override fun testAddDeadBreaks() {
-        super.testAddDeadBreaks()
+        testTransformationAndReduction(
```

Comment:
Please add a comment that trial and error was used to find a random seed that causes at least one injection to occur, and that this might need to be updated in future if the probabilities that control transformation change.